### DISCUSSION TITLE
How to add a new user through cluster operator

### DISCUSSION THREAD

**Discussion by charlienss** [rabbitmq/cluster-operator#1418] [Q&A] [answered]
How to add a new user through cluster operator:
When I started a Rabbitmq cluster using cluster operator, I found that there was only one default account. If I want to add custom users at startup, what should I do
How to add an account after I establish a cluster? I don't want to configure it through the interface. Can I directly add a user through the secret method? I think the cluster operator should have this function

**Comment by mkuratczyk** [comment, marked as answer]
You can use the Topology Operator, which gives you additional CRDs such as users, queues, etc:
https://www.rabbitmq.com/kubernetes/operator/using-topology-operator.html

Alternatively, you can rely on the definitions import feature https://www.rabbitmq.com/definitions.html:
https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/import-definitions
https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/import-https-url-definition

**Reply by charlienss** [reply]
Ok, thank you very much
